client/Packages/com.beamable.server/Editor/Usam/Editor/CodeService.cs

```diff
@@ -290,6 +290,52 @@ public async Promise CreateService(string name, ServiceType type, List<IBeamoSer
 			}
 		}
 
+		
+		public async Promise UpdateServiceReferences(string serviceName, List<string> assemblyReferencesNames)
+		{
+			LogVerbose($"Starting updating references");
+			//get a list of all references of that service
+			var service = _services.FirstOrDefault(s => s.name == serviceName);
+			if (service == null)
+			{
+				return;

```

Comment:
nitpick; but this pattern rubs me the wrong way; a silent return makes the eventual bug, "I passed an invalid serviceName", really really hard to find. Can we put a warning in here, or throw an exception? If it is _never_ okay to call this method with an invalid name, then we _shouldn't_ call it with an invalid name, and if we _are_, we should KNOW about it so we can fix it.

Reply:
agreed, just changed it for an `ArgumentException` instead